src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/results/results.html

```diff
@@ -96,9 +96,38 @@
                 </span>
               </p>
             </div>
+            
+            <div ng-if="record._source._extra_data.matches.fuzzy.length > 1 
+            && record._source._workflow.data_type === 'hep' && record._source._workflow.status === 'HALTED'"
+            class="matches-container">
+                <div ng-if="vm.record_matched.length < 1" >
```

Comment:
IMHO it will prevent other matches, ``vm.record_matched`` is a controller's variable and if a users click a match all the others will disappear